apps/frontend/src/libs/components/page-layout/styles.module.css

```diff
@@ -0,0 +1,41 @@
+:root {
+	--header-height: 73px;
```

Comment:
If --header-height is used only for the page and related components, it's better to keep it in .page scope, not in root

Reply:
oh, right, changed it